skins/laika/src/pages/donation_form.ts

```diff
@@ -34,7 +55,14 @@ new Vue( {
 		},
 	},
 	[
-		h( Component, {} ),
+		h( Component, {
+			props: {
+				paymentAmounts: pageData.applicationVars,
+				validateAmountURL: donationForm.getAttribute( 'data-validate-amount-url' ),
```

Comment:
When #1488 is merged, you can replace this with `pageData.applicationVars.urls.validateDonationAmount`